config/example_factories.go

```diff
@@ -325,11 +325,26 @@ func (f *ExampleProcessorFactory) CreateMetricsProcessor(
 	return nil, configerror.ErrDataTypeIsNotSupported
 }
 
-// RegisterTestFactories registers example factories. This is only used by tests.
-func RegisterTestFactories() error {
-	_ = receiver.RegisterFactory(&ExampleReceiverFactory{})
-	_ = receiver.RegisterFactory(&MultiProtoReceiverFactory{})
-	_ = exporter.RegisterFactory(&ExampleExporterFactory{})
-	_ = processor.RegisterFactory(&ExampleProcessorFactory{})
-	return nil
+// TestComponents registers example factories. This is only used by tests.
+func TestComponents() (
```

Comment:
Can you use a prefix that is different from "Test"? It is a bit confusing with this prefix, looks like a test.

Reply:
`BuildTestComponents`?

Reply:
I renamed to `ExampleComponents()` since the file is called `example_factories.go`